tools/releaseBuild/azureDevOps/AzArtifactFeed/PSGalleryToAzArtifacts.yml

```diff
@@ -0,0 +1,32 @@
+# Sync packages from PSGallery to Azure DevOps Artifacts feed
+
+resources:
+- repo: self
+  clean: true
+
+queue:
+  name: Hosted VS2017
+steps:
+  - powershell: |
+      Install-Module -Name PowerShellGet -MinimumVersion 2.0.1 -Force
+      Import-Module PowerShellGet -Force -Verbose
+    displayName: Update PSGet and PackageManagement
+    condition: succeededOrFailed()
+
+  - powershell: |
+      Import-Module -Force "$(Build.SourcesDirectory)/tools/releaseBuild/azureDevOps/AzArtifactFeed/SyncGalleryToAzArtifacts.psm1"
+      SyncGalleryToAzArtifacts -AzDevOpsPAT $(AzDevOpsPAT) -Destination $(Build.ArtifactStagingDirectory)
+    displayName: Download packages from PSGallery that need to be updated
+    condition: succeededOrFailed()
+
+  - powershell: |
+      Write-Verbose -Verbose "Packages to upload"
+      (Get-ChildItem "$(Build.ArtifactStagingDirectory)/*.nupkg").FullName
+    displayName: List packages to upload
+    condition: succeededOrFailed()
+
+  - task: NuGetCommand@2
```

Comment:
As we discussed; this needs to be double-checked.

Reply:
Updated to use the PAT